### ISSUE TITLE
Debugging not working as described in the documentation

### ISSUE THREAD

**Issue by ascheucher-shopify-partner** [sst/sst.dev#766]
Trying to figure out an issue I have the SST Guide, chapter *[Create a settings page](https://sst.dev/chapters/create-a-settings-page.html)* sub chapter *[Connecting the billing form](https://sst.dev/chapters/connect-the-billing-form.html)* I tried to follow the *[How to debug lambda functions in VSC](https://sst.dev/examples/how-to-debug-lambda-functions-with-visual-studio-code.html), but there are some issues:

* I had to change in the launch.json to `"runtimeArgs": [dev"]`
* I didn't find an equivalent to *--increase-timeout*
* I can't set break points. VSC gives me this warning: `Unbound breakpoint - Some of your breakpoints could not be set. If you're having an issue, you can troubleshoot your launch configuration`
* Warning: You are using a global installation of SST but you also have a local installation specified in your package.json. The local installation will be used but you should typically run it through your package manager.

I assume this is due to the changes from v2 to v3, but how can I debug now?

**Comment by jayair** [comment]
Yeah that VSCode chapter is out of date, I'm going to archive it soon.

As for breakpoint support, we are working on it: https://github.com/sst/ion/issues/73

**Comment by ascheucher-shopify-partner** [comment]
Thanks for the pointer!
